operators/multiclusterobservability/manifests/base/config/metrics_allowlist.yaml

```diff
@@ -356,3 +356,40 @@ data:
         expr: sum(rate(node_network_receive_drop_total{job=\"node-exporter\", device!=\"lo\"}[1m])) without(device)
       - record: instance:node_network_transmit_drop_excluding_lo:rate1m
         expr: sum(rate(node_network_transmit_drop_total{job=\"node-exporter\", device!=\"lo\"}[1m])) without(device)
+
```

Comment:
Are UWL metrics &  recording rules control plane metrics to scrape from a hypershift guest cluster?